bigfeta/schemas.py

```diff
@@ -337,6 +337,11 @@ class BigFetaSchema(ArgSchema):
             description=("tilespec.tforms[i].tform() for i in transform_apply "
                          "will be performed on the matches before matrix "
                          "assembly."))
+    solve_implementation = String(
+        required=False, default="default", missing="default",
+        validator=mm.validate.OneOf(["petsc", "python", "default"]),
```

Comment:
since they are both python front-ends now, would this make more sense to call then `petsc` and `scipy`?

Reply:
fair point -- I will change this.